registrations/serializers.py

```diff
@@ -59,7 +41,50 @@ def create(self, validated_data):
             signup.save()
             return signup
         else:
-            raise DRFPermissionDenied("The waiting list is already full")
+            raise DRFPermissionDenied(_("The waiting list is already full"))
```

Comment:
No need to change this. Just thinking if this should be also turned into a validation error in the future, so that a proper error could be rendered by the UI more easily. :thinking:

Reply:
That's definitely something the could be done later.